texts/epn_paper.md

```diff
@@ -0,0 +1,74 @@
+# Let's talk online
+
```

Comment:
I would like an explicit approval from everyone listed: @xabomon @vfatemi @greplova @everthemore @babaks77 @dvarjas.

```suggestion
## Authors (in alphabetical order)

- Anton Akhmerov (Delft University of Technology)
- Xavier Bonet-Monroig (Leiden University)
- Valla Fatemi (Yale University)
- Eliska Greplova (ETH Zürich)
- Evert van Nieuwenburg (University of Copenhagen)
- Babak Seradjeh (Indiana University)
- Daniel Varjas (Delft University of Technology)
```

Reply:
👍

Reply:
Sorry for my little contribution on the writing. I'm OK with either being included or not.
Thanks for the effort, it looks great!

Reply:
👍

Reply:
Still waiting for @vfatemi @dvarjas @babaks77

Reply:
:+1:

Reply:
Approved